cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Ensure that the `urls.py` files in the `cinema` and `user` apps define an `app_name` attribute to support the use of namespaces in the `include` function. This is necessary for the `namespace` argument to work correctly.